Doc/using/configure.rst

```diff
@@ -964,9 +964,18 @@ Main Makefile targets
   You can use the configure :option:`--enable-optimizations` option to make
   this the default target of the ``make`` command (``make all`` or just
   ``make``).
-* ``make buildbottest``: Build Python and run the Python test suite, the same
-  way than buildbots test Python. Set ``TESTTIMEOUT`` variable (in seconds)
-  to change the test timeout (1200 by default: 20 minutes).
+
+* ``make test``: Build Python and run the Python test suite with ``--slow-ci``
```

Comment:
This doc is incorrect, this PR made `test` use `--fast-ci`.

Reply:
Oops, I included a fix in my following PR: gh-109903.